src/electronic-ids/pcsc/LatEIDIDEMIAv2.cpp

```diff
@@ -63,7 +63,7 @@ byte_vector LatEIDIDEMIAV2::getCertificateImpl(const CertificateType type) const
     auto info =
         readDCODInfo(CERT_FILE_REF, type.isAuthentication() ? data->authCache : data->signCache);
     if (TLV id = TLV::path(info, 0x30, 0xA1, 0x30, 0x30, 0x04)) {
-        return electronic_id::getCertificate(*card, CommandApdu::select(0x02, {id.begin, id.end}));
+        return readFile(*card, CommandApdu::selectEF(0x02, {id.begin, id.end}), 0xC0);
```

Comment:
Perhaps add a comment that Idemia does not support 256 byte block length, need to limit the block length to 0xC0.